config/environments/production.rb

```diff
@@ -110,4 +110,7 @@
   # config.active_record.database_selector = { delay: 2.seconds }
   # config.active_record.database_resolver = ActiveRecord::Middleware::DatabaseSelector::Resolver
   # config.active_record.database_resolver_context = ActiveRecord::Middleware::DatabaseSelector::Resolver::Session
+
+  # this needs to be updated before production :host should be set to the actual host of your application.
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```

Comment:
to nie jest poprawne. Production.rb odpowiada za srodowisko produkcyjne, tj. "online" - tam nie bedzie ani portu 3000 ani takiego hosta :). Docsy devise mowia, ze nalezy to dodac tylko do development.rb